src/content/docs/baas/Firebase/firebase-case-study-nikolaos-digalakis.md

```diff
@@ -0,0 +1,266 @@
+# Case Study: Google Firebase
+
+## Introduction
+
```

Comment:
I think the topic should be reworded to focus more on QuickTask. You may consider something like `Quick Task a case study of Firebase usage`